olive/evaluator/olive_evaluator.py

```diff
@@ -415,81 +413,83 @@ def get_inference_settings(cls, metric: Metric, model: ONNXModelHandler) -> Dict
         if model_infrerence_settings:
             inference_settings.update(model_infrerence_settings)
 
-        metric_inference_settings = metric.get_inference_settings(cls.framework.lower())
+        metric_inference_settings = metric.get_inference_settings(Framework.ONNX.lower())
         if metric_inference_settings:
             inference_settings.update(metric_inference_settings)
 
         return inference_settings
 
-    def _inference(
-        self,
+
+class OnnxEvaluator(OliveEvaluator, OnnxEvaluatorMixin, framework=Framework.ONNX):
+    def __init__(self):
```

Comment:
needless?

Reply:
Which part? the `__init__`? I can remove it but kept it since the original code also had it.